src/application-insights/azext_applicationinsights/_help.py

```diff
@@ -21,6 +21,126 @@
           provided, then --offset will be ignored.
 """
 
+helps['monitor app-insights component'] = """
+    type: group
+    short-summary: Manage an Application Insights component or its subcomponents.
+"""
+
+helps['monitor app-insights component create'] = """
+    type: command
+    short-summary: Create a new Application Insights resource.
+    parameters:
+      - name: --application-type
+        type: string
+        short-summary: Type of application being monitored. Possible values include 'web', 'other'. Default value is'web' .
+      - name: --kind -k
+        type: string
+        short-summary: The kind of application that this component refers to, used to customize UI. This value is a freeform string, values should typically be one of web, ios, other, store, java, phone.
+    examples:
+      - name: Create a component with kind web and location.
+        text: |
+          az monitor app-insights component create --app demoApp --location westus2 --kind web -g demoRg --application-type web
+"""
+
+helps['monitor app-insights component update'] = """
+    type: command
+    short-summary: Update properties on an existing Application Insights resource. The primary value which can be updated is kind, which customizes the UI experience.
+    parameters:
+      - name: --kind -k
+        type: string
+        short-summary: The kind of application that this component refers to, used to customize UI. This value is a freeform string, values should typically be one of web, ios, other, store, java, phone.
+    examples:
+      - name: Update a component with kind web.
+        text: |
+          az monitor app-insights component update --app demoApp -k web -g demoRg
+"""
+
+helps['monitor app-insights component update-tags'] = """
+    type: command
+    short-summary: Update tags on an existing Application Insights resource.
+    examples:
+      - name: Update the tag 'name' to equal 'value'.
+        text: |
+          az monitor app-insights component update-tags --app demoApp --tags name=value -g demoRg
+"""
+
+helps['monitor app-insights component show'] = """
```

Comment:
I suggest add a command of `monitor app-insights component list` for `List components in a resource group or subscription`

Reply:
added